@xen-orchestra/web/src/components/vm/VmVirtualisationAndBoot.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <UiCard class="vm-virtualisation-and-boot">
```

Comment:
```suggestion
  <UiCard class="vm-virtualization-and-boot">
```

don't forget to change the name component too